### ISSUE TITLE
Auto-Type interception window with search bar

### ISSUE THREAD

**Issue by sandreas** [MacPass/MacPass#1224]
I'm always frustrated when I press the Auto-Type option and MacPass does not find any match / matches that are invalid.

Best case scenario would be that if the autotype does not find a unique match, an interception window would appear, where it is possible to select AND/OR search for the wanted database item.

KeepassXC already offers this interception window, but does not offer a search bar for all items, which I would really appreciate. Related to this I added a feature request to their project containing this screenshot - see https://github.com/keepassxreboot/keepassxc/issues/7654 for details:
![keepassxc_screenshot](https://user-images.githubusercontent.com/2050604/160201095-567d2459-e608-4bbe-b0ae-0142ca240e9e.png)

**Comment by georgesnow** [comment]
I totally would love that! I have wanted it for ages.

**Comment by sandreas** [comment]
BTW: I was wrong. With 2.7.0 KeepassXC has this feature and it works great on Linux... I'd still prefer using MacPass on macOS providing a native look and feel.

**Comment by mstarke** [comment]
Was about to comment that 2.7 did I introduce this. I was pondering about something like this as well. Also @georgesnow did suggest a similar idea a while ago. I’m currently reworking the inspector so this will have to take a back seat until I have finished the inspector changes.

**Comment by sandreas** [comment]
>  I’m currently reworking the inspector so this will have to take a back seat until I have finished the inspector changes.

Cool, thank you. I'm not in a hurry. Unfortunately I don't know Cocoa/SWIFT well enough to help you out, so good luck :-)

**Comment by georgesnow** [comment]
Yeah, I had taken a stab at it a while ago. I never got something workable. That’s how and why my plug-in was actually born.
